app/styles/global.css

```diff
@@ -154,6 +167,12 @@ body {
 .width-18 {
   width: 18rem;
 }
+.max-width-15 {
```

Comment:
each of these attributes should be a separate class

Reply:
done!